src/west/configuration.py

```diff
@@ -130,6 +130,19 @@ def delete(self, option: str):
             self.cp.write(f)
 
 
+class _Converter:
```

Comment:
I miss the value of wrapping two small static methods in this new class. Why not just functions? Or inside some existing class.


https://steve-yegge.blogspot.com/2006/03/execution-in-kingdom-of-nouns.html

Reply:
> Why are just functions better than functions within a (private) class?

Because every piece of code must have some purpose? The purpose of a class is to define objects that combine data (fields) and code using that data (methods), that's OoO in a nutshell. If there's no data and just two static methods, why is there a class? This is not Java, classes are not mandatory. BTW top-level functions is one of the many things Kotlin fixed Java: https://kotlinlang.org/docs/functions.html

I have not really thought about what is the best place for these functions, but their _Converter "container" seemed pointless.

> Sorry, but I don't see that they belong to class _InternalCF which says

Maybe I missed something but it looked like these functions were only used with `_InternalCF` data.